src/System.Private.CoreLib/shared/System/IntPtr.cs

```diff
@@ -23,7 +23,7 @@ public struct IntPtr : IEquatable<IntPtr>, ISerializable
         // See https://github.com/dotnet/corert/blob/master/Documentation/design-docs/diagnostics/diagnostics-tools-contract.md for more details. 
         // Please do not change the type, the name, or the semantic usage of this member without understanding the implication for tools. 
         // Get in touch with the diagnostics team if you have questions.
```

Comment:
Me too. Would the existing automated test suites catch this if it were a breaking change?

Reply:
The existing tests may not be sufficient to catch the issues. Changes like this introduce new combinations that the existing tests were not designed to exercise.

We have tried to mark `Nullable<T>` as readonly. All existing tests were green on the change. But it had to be rolled back because of it was identified as breaking change later. https://github.com/dotnet/corefx/pull/24997#issuecomment-346523578

Reply:
Interesting! Although it appears System.Nullable<T> may have been a special case.

However, I defer to your more qualified judgement in dropping this PR.